src/Inciter/Transporter.h

```diff
@@ -426,28 +426,14 @@ class Transporter : public CBase_Transporter {
 
     //! Non-reduction target for receiving progress report on flattening mesh
     void peflattened() { m_progReorder.inc<0>(); }
+    //! Non-reduction target for receiving progress report on node ID query
+    void pequery() { m_progReorder.inc<1>(); }
     //! Non-reduction target for receiving progress report on node ID mask
-    void pemask() { m_progReorder.inc<1>(); }
```

Comment:
Is this another location we can try and replace magic numbers with some semantic content?

Reply:
Yeah, you are right, I could do an enum or something. Noted, will fix.

Reply:
And after pushing a new commit, my previous reply to this is, of course, gone! Hey @rfbird, I hate to be a pain, but let's not use github for code review anymore, reviewable.io is a lot nicer, and most importantly, preservers our discussion.